docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,512 @@
+processing is complete, so make sure that you download the files within that two-week window.**
+
+We plan to support adding standard OPERA RTC-S1 products generated On Demand to the OPERA_L2_RTC-S1_V1 collection, 
+expanding the archive based on user needs. Stay tuned for that functionality!
+
+## Product Packaging
+
+The products generated by the OPERA SDS are available for download as individual files associated with the 
+OPERA_L2_RTC-S1_V1 record corresponding to the Sentinel-1 burst used as input for the product.
+
+Products generated On Demand using ASF's HyP3 platform are delivered as zip files that include all the files 
+available for the products generated by the OPERA SDS. The zip file also includes the color browse image (which is 
+not georeferenced) used for displaying archived OPERA RTC-S1 products in Vertex. 
+
+The naming convention for OPERA RTC-S1 products generated On-Demand using ASF's HyP3 platform is the same as for 
+the OPERA RTC-S1 products generated by the OPERA SDS.
+
+### L2 Radiometric Terrain Corrected (RTC) Files
+OPERA RTC-S1 products are available as a collection of files associated with a source Sentinel-1 burst. If you search 
+for OPERA-S1 products in Vertex, the results for each Sentinel-1 burst provide access to a number of files. If you 
+click on an item in the left panel of the search results, the associated files are listed in the right panel. 
+
+![Figure 3](../images/opera-rtc-vertex-results.png "OPERA RTC-S1 Search Results")
+
+The files available for download include: 
+
+- HDF5 file containing product metadata, specifically orbit position and velocity (no actual RTC data are 
+  included in this file)
+- A single-band 32-bit float Cloud-Optimized GeoTIFF (COG) file for each available polarization containing the 
+  RTC values (most commonly VV and VH, but HH and HV in some areas)
+- [Mask](#validity-mask) COG file indicating pixels in the RTC products that contain valid data and indicating 
+  which pixels are impacted by layover and/or shadow
+- Metadata XML file containing information about the product in ISO format
+- [Local Incidence Angle](#local-incidence-angle) COG file 
+
+### L2 Radiometric Terrain Corrected Static Layer (RTC-STATIC) Files
+There are some ancillary products required for RTC processing that change very little through time. Instead of 
+re-generating these reference products for each Sentinel-1 burst acquisition, they are generated once for each 
+Sentinel-1 burst ID used for OPERA processing, and archived as `Static Layer` files.
+
+These `RTC-STATIC` products include the following COG files: 
+
+- [local incidence angle](#local-incidence-angle)
+- incidence angle
+- [mask](#validity-mask) (layover/shadow validity mask)
+- number of looks
+- RTC Area Normalization Factor (ANF) gamma0 to beta0
+- RTC ANF gamma0 to sigma0
+
+You can access the RTC-STATIC files the same way that you would the RTC-S1 files. Applying a `File Type` filter of 
```

Comment:
tiny style change. Feel free to ignore. 
```suggestion
You can access the RTC-STATIC files the same way you would the RTC-S1 files. Applying a `File Type` filter of 
```

Reply:
I've gone back and forth on this one. I think I'm just going to leave it.